src/pe_reports/report_generator.py

```diff
@@ -0,0 +1,42 @@
+"""A tool for creating Posture & Exposure reports."""
+
+# Standard Python Libraries
+import os
+import sys
+
+# Third-Party Libraries
+from _version import __version__
```

Comment:
This needs to be a relative import.

```suggestion
from ._version import __version__
```